Util/src/main/java/io/deephaven/util/SaturationArithmetic.java

```diff
@@ -0,0 +1,25 @@
+package io.deephaven.util;
+
+// See http://locklessinc.com/articles/sat_arithmetic/
+
+public class SaturationArithmetic {
+    public static long addSaturated(final long x, final long y) {
+        final long res = x + y;
+        if (res < x) {
+            return Long.MAX_VALUE;
+        }
+        return res;
+    }
+
+    public static long minusSaturated(final long x, final long y) {
+        final long res = x - y;
+        if (res <= x) {
+            return Long.MIN_VALUE;
+        }
+        return res;
+    }
```

Comment:
We agreed that this is wrong, and since it's also unused, that it should be deleted.